packages/table/components/CheckboxTable.tsx

```diff
@@ -7,13 +7,21 @@ export interface CheckboxTableProps extends TableProps {
   /**
    * Callback for when a user makes a selection. Passes selected row data as a parameter
    */
-  onChange?: (selectedRows: object[]) => void;
+  onChange?: (selectedRows: string[]) => void;
+  /**
+   * The unique key in each row's data object
+   */
+  uniqueKey: string;
 }
 
 class CheckboxTable extends React.PureComponent<CheckboxTableProps> {
   constructor(props) {
     super(props);
 
+    this.state = {
+      selected: props.selectedRows || []
```

Comment:
I don't see we're using this piece of state

Reply:
We're not, I forgot to delete it after testing something